src/main/webapp/js/finscholar/collegepageview/collegepageview.js

```diff
@@ -16,15 +16,18 @@
 
 goog.module('finscholar.collegepageview');
 
-const GoogDom = goog.require('goog.dom');
+const {SinglePageView} = goog.require('finscholar.singlepageview');
 const {loadCollegeData} = goog.require('datahandlers.collegepage');
 
 /** Class for the college page view. */
-class CollegePageView {
-  constructor() {}
+class CollegePageView extends SinglePageView {
+
+  constructor() {
+    super(undefined, undefined);
```

Comment:
currently CollegeDataHandler is not a class and it directly renders college page to view. So the call to parent here is just a place holder. We'll think about overall structure of our code more during alpha.

Reply:
I'll be working off this branch today, should I make CollegeDataHandler into a class?